xc7/tests/srl/srl32_init/sim/run_simulation.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+iverilog -c sim_sources.cf -s tb -DSIMULATION -o tb.vvp -v
```

Comment:
Use add_fpga_target's TESTBENCH_SOURCES to run the simulation rather than a shell script.  This adds targets to run the simulation and visualize the wave output.

Reply:
Yep. I removed this script and added TESTBENCH_SOURCES for all test cases.